src/types.ts

```diff
@@ -43,6 +43,8 @@ export interface Column<TRow, TSummaryRow = unknown> {
   readonly resizable?: Maybe<boolean>;
   /** Enable sorting of a column */
   readonly sortable?: Maybe<boolean>;
+  /** Enable dragging of a column */
+  readonly draggable?: Maybe<boolean>;
```

Comment:
I wonder if this is the best API for this feature, what if we want some columns to never move? i.e. are there use cases where we'd want columns to move only in a portion of the grid.

Reply:
I think for now users can decide whether to move a column or not. We can make `frozen` columns not draggable but there may be a use cases where frozen columns needs to reordered